usagov/usagov-2021
# USAGOV-1683 benefits search title

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1683

## Description

Updates views used on benefits search page to pull in the new search title field for life events
Updates templates to use search title field instead of raw title field for life events
Fixes a bug I ran into if more than category referenced a life event, that event would show up once for each category.

## Type of Changes

  - Search title field (if set) used in life events grid
![image](https://github.com/usagov/usagov-2021/assets/493854/adc7daab-bb9d-454b-a4c7-17396b1f34b0)
  - Search title field (if set) used in search results
![image](https://github.com/usagov/usagov-2021/assets/493854/7f859ba4-e704-4bf3-9529-0bd0eaf4bc75)


## Testing Instructions


Assumes you have english/spanish benefits search pages set up.
Also need to have one or more benefit categories reference some life events (edit benefits category terms and update the life events field)
Update the life events referenced so that the benefits search title field is not empty (if empty, the title field will be used as a fallback)
 
### Requires New Config

### Requires New Content

### Validation Steps
- Update english and spanish content per testing steps above
- On English search page, pick one or more categories and select at least one that has a lifetime event with the search title field populated. You should see that title used in the search results listing when you search. You should also see that search title used in the life event grid at the top of the page.
- On Spanish search page, pick one or more categories and select at least one that has a lifetime event with the search title field populated. You should see that title used in the search results listing when you search. You should also see that search title used in the life event grid at the top of the page.

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
